amt/api/routes/project.py

```diff
@@ -35,3 +37,123 @@ async def get_root(
         raise NotFound()
 
     return templates.TemplateResponse(request, "pages/index.html.j2", context)
+
+
+# !!!
+# Implementation of this endpoint is for now independent of the project ID, meaning
+# that the same system card is rendered for all project ID's. This is due to the fact
+# that the logical process flow of a system card is not complete.
+# !!!
+@router.get("/{project_id}/system_card")
+async def get_system_card(
+    request: Request,
+    project_id: int,
+    projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
+    file_system_storage_service: Annotated[FileSystemStorageService, Depends(FileSystemStorageService)],
+) -> HTMLResponse:
+    logger.info(f"getting project with id {project_id}")
+    project = projects_service.get(project_id)
+    if not project:
+        logger.warning(f"project with id {project_id} not found")
+        raise NotFound()
+
+    file_system_storage_service.base_dir = Path("example_system_card/")
```

Comment:
See remark about the function dependency and use StorageFactory.init(storage_type="file", location=path.parent, filename=path.name) instead.